lti_consumer/lti_1p3/constants.py

```diff
@@ -34,6 +34,32 @@
     ],
 }
 
+# Context membership roles
```

Comment:
@shimulch We already have a role map (right above, `LTI_1P3_ROLE_MAP`).

For the context membership service, we don't need the "simple" version (it's just used in the `Resource Link Membership Service`). 

Can you remove this and simplify the current implementation?

Reply:
@giovannicimolin Are ``institution roles`` can be used interchangeably with ``context roles``? IMS seems to have separate vocabulary for both of them.

Reply:
@shimulch I've missed that these are context roles as opposed of institution roles. You can keep this dict as is.